src/main/webapp/courseLoader.js

```diff
@@ -380,77 +380,4 @@
       createComment(comment, isCourse);
     }
   }
-
```

Comment:
This is all content we do want to delete right?

Reply:
Yes! It is still being used, I just wanted to separate mine and Dagmawi's stuff in different files.